src/componentLibrary/Table/Table.tsx

```diff
@@ -0,0 +1,36 @@
+import { ReactNode } from 'react';
+
+import { StyledTable, TableWrapper, TableRow, TableHeader, TableCell } from './Table.style';
+
+interface TableProps {
+  columnLabels: string[];
+  rows: ReactNode[][];
```

Comment:
` ReactNode[][]` looks a bit bit funky :) anything we can do to tidy it up?

Reply:
Good catch, I added type for row :)